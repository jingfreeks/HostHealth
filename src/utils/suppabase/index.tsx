export {default as supabase} from './suppabase';
