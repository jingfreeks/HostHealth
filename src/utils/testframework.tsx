import React from 'react';
import {Provider} from 'react-redux';
import configureStore from 'redux-mock-store';
import thunk from 'redux-thunk';
// import {QueryClient, QueryClientProvider} from '@tanstack/react-query';
// // import {DARK_THEME} from '@/utils/constants';
// // import {ThemeProvider} from 'styled-components/native';

type MockProviderType = {
  children?: React.ReactNode;
  store?: object;
};
export const MockProvider = (props: MockProviderType) => {
  const {store, children} = props;
  const mockStore = configureStore([thunk]);
  const mocked = mockStore(store);
  return <Provider store={mocked}>{children}</Provider>;
};

export const testingProps = (reference: string): unknown => ({
  testID: reference,
  accessibilityLabel: reference,
});

// const queryClient = new QueryClient({
//   defaultOptions: {
//     queries: {
//       retry: false,
//     },
//   },
// });
// export const wrapper = ({children}) => (
//   <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
// );
