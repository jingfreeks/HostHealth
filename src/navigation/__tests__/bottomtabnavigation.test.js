import * as React from 'react';
import {renderWithProviders} from '@/utils/testframeworknew';
import Bottomtabnavigation from '../bottomtabnavigation';


test('Bottomtabnavigation', () => {
  const all = renderWithProviders(<Bottomtabnavigation />);
  expect(all.toJSON()).toMatchSnapshot();
});
