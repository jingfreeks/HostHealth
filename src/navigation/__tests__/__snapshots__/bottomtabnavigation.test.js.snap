// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bottomtabnavigation 1`] = `null`;
